From b466e8332eafdb8dda2cd8a7879dd69638d0118a Mon Sep 17 00:00:00 2001
From: chewitt <github@chrishewitt.net>
Date: Fri, 12 Jul 2019 20:37:37 +0000
Subject: [PATCH 173/182] WIP: arm64: dts: meson-gxm-khadas-vim2: use rc-khadas
 remote

Khadas shipped the first batch of VIM devices with the 'geekbox' remote from
another of their brands. All further shipments have used a 'khadas' branded
remote that provides an Android mouse function instead of KEY_SCREEN. As the
IR keycode for the geekbox screen button is different to the khadas mouse
button KEY_SCREEN doesn't work on the Khadas remote, so swap to the rc-khadas
keymap to gain an extra button mapped to KEY_MUTE.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts b/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts
index 150a6ce..0872756 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts
@@ -336,7 +336,7 @@
 	status = "okay";
 	pinctrl-0 = <&remote_input_ao_pins>;
 	pinctrl-names = "default";
-	linux,rc-map-name = "rc-geekbox";
+	linux,rc-map-name = "rc-khadas";
 };
 
 &pwm_AO_ab {
-- 
2.7.4

